src/components/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = (props) => {
+  const { name } = props;
```

Comment:
[Optional]

We can destructure the `name` prop from the parameter bracket directly since we won't be using `props` anywhere else.

```suggestion
const Button = ({ name }) => {
```